### PR TITLE
ci: enable CI tools and linters

### LINKED ISSUE #166: Use CI tools like `changesets` for changelog and releasing
https://github.com/changesets/changesets

Related bot https://github.com/apps/changeset-bot and action https://github.com/changesets/action

An example: https://github.com/import-js/eslint-import-resolver-typescript/pull/118/files

### PR DESCRIPTION
**Pull request by JounQin**
close #166

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ title: "eslint-plugin-svelte"
...
-[![NPM downloads](https://img.shields.io/badge/dynamic/json.svg?label=downloads&colorB=green&suffix=/day&query=$.downloads&uri=https://api.npmjs.org//downloads/point/last-day/eslint-plugin-svelte&maxAge=3600)](http://www.npmtrends.com/eslint-plugin-svelte)
+[![NPM downloads](https://img.shields.io/badge/dynamic/json.svg?label=downloads\&colorB=green\&suffix=/day\&query=$.downloads\&uri=https://api.npmjs.org//downloads/point/last-day/eslint-plugin-svelte\&maxAge=3600)](http://www.npmtrends.com/eslint-plugin-svelte)
 [![NPM downloads](https://img.shields.io/npm/dw/eslint-plugin-svelte.svg)](http://www.npmtrends.com/eslint-plugin-svelte)
 [![NPM downloads](https://img.shields.io/npm/dm/eslint-plugin-svelte.svg)](http://www.npmtrends.com/eslint-plugin-svelte)
 [![NPM downloads](https://img.shields.io/npm/dy/eslint-plugin-svelte.svg)](http://www.npmtrends.com/eslint-plugin-svelte)
 [![NPM downloads](https://img.shields.io/npm/dt/eslint-plugin-svelte.svg)](http://www.npmtrends.com/eslint-plugin-svelte)
 [![Build Status](https://github.com/ota-meshi/eslint-plugin-svelte/workflows/CI/badge.svg?branch=main)](https://github.com/ota-meshi/eslint-plugin-svelte/actions?query=workflow%3ACI)
```
Comment (JounQin) [inline docs/README.md:18]: Hah, some new badges are missing here.

File: docs/README.md [hunk 2]
```diff
@@ -4,22 +4,22 @@ title: "eslint-plugin-svelte"
 
 # Introduction
 
-`eslint-plugin-svelte` is [ESLint] plugin for [Svelte].  
-It provides many unique check rules by using the template AST.  
+`eslint-plugin-svelte` is [ESLint] plugin for [Svelte].\
```
Comment (JounQin) [inline docs/README.md:7]: It could be a bit of noise, I'll remove them.

File: README.md [hunk 3]
```diff
@@ -238,14 +245,15 @@ Example **.vscode/settings.json**:
...
 <!--USAGE_SECTION_END-->
 
 ## :white_check_mark: Rules
 
 <!--RULES_SECTION_START-->
 
-:wrench: Indicates that the rule is fixable, and using `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the reported problems.  
-:bulb: Indicates that some problems reported by the rule are manually fixable by editor [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).  
+:wrench: Indicates that the rule is fixable, and using `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the reported problems.
+:bulb: Indicates that some problems reported by the rule are manually fixable by editor [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).
```
Comment (ota-meshi) [inline README.md:256]: I want a line break here. Can we leave trailing spaces?

File: README.md [hunk 4]
```diff
@@ -254,81 +262,82 @@ Example **.vscode/settings.json**:
...
-| [svelte/no-not-function-handler](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-not-function-handler/) | disallow use of not function in event handler | :star: |
-| [svelte/no-object-in-text-mustaches](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-object-in-text-mustaches/) | disallow objects in text mustache interpolation | :star: |
-| [svelte/no-shorthand-style-property-overrides](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-shorthand-style-property-overrides/) | disallow shorthand style properties that override related longhand properties | :star: |
-| [svelte/no-unknown-style-directive-property](https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-unknown-style-directive-property/) | disallow unknown `style:property` | :star: |
-| [svelte/valid-compile](https://ota-meshi.github.io/eslint-plugin-svelte/rules/valid-compile/) | disallow warnings when compiling. | :star: |
+| Rule ID                                                                                                                                       | Description                                                                   |                |
```
Comment (ota-meshi) [inline README.md:265]: This table is automatically generated by `yarn update`. So I think the script needs to be changed as well.

Comment (JounQin) [reply README.md:265]: They can be ignored. I'll update this PR.

File: README.md (hunk 3 above)
Comment (JounQin) [reply README.md:256]: Sure.

**Review by ota-meshi — APPROVED** [review]
Thank you!

**Pull request merged**